Framework/Json/src/Json.cpp

```diff
@@ -11,6 +11,26 @@
 
 namespace Mantid::JsonHelpers {
 
+void replaceAll(std::string &str, const std::string &from, const std::string &to);
```

Comment:
The function signature (L14) can go away and the function itself should move into an anonymous namespace so the compiler knows to not publicly share it

Reply:
Just did the correction per your suggestion